config/default.yml

```
@@ -83,6 +83,15 @@ Rails/ActiveSupportAliases:
   Enabled: true
   VersionAdded: '0.48'
 
+Rails/AddColumnComment:
+  Description: >-
+    This cop enforces the use of the `comment` option when creating a new column
+    during a migration by calling either `add_column`, `t.column` or any other
```

```suggestion
    during a migration by calling `add_column`, `t.column` or any other
```
'either' would imply there are only two choices.